pkg/imgpkg/cmd/copy_repo_src_test.go

```diff
@@ -615,7 +620,7 @@ func TestToRepoBundleCreatesValidLocationOCI(t *testing.T) {
 			APIVersion: "imgpkg.carvel.dev/v1alpha1",
 			Kind:       "ImageLocations",
 			Images: []bundle.ImageLocation{{
-				Image:    "index.docker.io/library/hello-world@sha256:ebf526c198a14fa138634b9746c50ec38077ec9b3986227e79eb837d26f59dc6",
+				Image:    dockerhubProxy + "/library/hello-world@sha256:ebf526c198a14fa138634b9746c50ec38077ec9b3986227e79eb837d26f59dc6",
```

Comment:
This maybe can be a call similar to:
```go
env.CompleteImageRef("hello-world@sha256:ebf526c198a14fa138634b9746c50ec38077ec9b3986227e79eb837d26f59dc6")
```
This function would add the `index.docker.io` or the proxy depending on the environment variable.

I am suggesting this because it feels like sprinkling this logic in different tests makes it harder to follow as a pattern while if we do this as part of a function call it might be easier to understand.
While I was writing this an idea came to my head, maybe the registry builder could be used to make this translation for us because in that case it would be transparent to the person writing the test and would be applicable to all the tests with a change in a single place (I did not validate the feasibility of this, it is just an idea if you want to explore it)

Reply:
Yes, good idea. Need to give it a thought how to make it transparent.